omrsigcompat/makefile

```diff
@@ -25,6 +25,20 @@ OBJECTS := omrsig
 OBJECTS := $(addsuffix $(OBJEXT),$(OBJECTS))
 EXPORT_FUNCTIONS_FILE := omrsig.exportlist
 
+ifeq (win,$(OMR_HOST_OS))
+COPYRIGHT_RC_FILE := ../../omrsigcompat/omrsig.rc
+COPYRIGHT_RES_FILE := omrsig.res
```

Comment:
1) where do these files come from?
omrsig.rc is already created outside of OMR project in a folder with the same name (e.g. ../../**omrsigcompat** in our case). We assume other users also create omrsig.rc at the same
path. They could change the path if omrsig.rc is created somewhere else.

omrsig.rc is used here to generate omrsig.res at omr/omrsigcompat where it is linked to omrsig.dll so as to add version information in there.

2) Is this sort of DLL copyright information added to other OMR DLLs
It depends whether the .dll file is exposed to a product. For now, it seems only omrsig.dll is captured in such case.